doc/nrf/known_issues.rst

```diff
@@ -1124,6 +1124,20 @@ Subsystems
 Bluetooth® LE
 =============
 
+.. rst-class:: v2-0-0
+
+NCSDK-15527: Advertising in :ref:`peripheral_hr_coded` and scanning in :ref:`bluetooth_central_hr_coded` cannot be started when using the SW Split Link Layer.
+  The :kconfig:option:`CONFIG_BT_CTLR_ADV_EXT` option required by these samples is disabled by default in the SW Split Link Layer.
+
+  **Workaround:** Enable the :kconfig:option:`CONFIG_BT_CTLR_ADV_EXT` option in the project configuration file (:file:`prj.conf`).
+
+  .. rst-class:: v2-0-0 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0
+
+NCSDK-15229: Incorrect peer's throughput calculation in the :ref:`ble_throughput` sample.
+  The peer's measured throughput is understated because it includes a delay, where no data transfer is done.
+
+  **Workaround:** Manually cherry-pick and apply commit with fix from main (commit hash: ``2ea62e3ac5c8164046643cf13e45b808d3ebab43``).
```

Comment:
DNM till https://github.com/nrfconnect/sdk-nrf/pull/7886 is merged so correct SHA can be put here.